cms/templates/js/course_info_update.underscore

```diff
@@ -11,8 +11,14 @@
 		</div>
 		<div class="row">
 			<!-- cid rather than id b/c new ones have cid's not id's -->
-			<a href="#" class="save-button" name="<%= updateModel.cid %>">Save</a>
+			<a href="#" class="save-button" name="<%= updateModel.cid %>">Post</a>
 			<a href="#" class="cancel-button" name="<%= updateModel.cid %>">Cancel</a>
+            <%if (push_notification_enabled) { %>
+                <div class="new-update-push-notification">
+                    <label class="inline-label">Send push notification:</label>
+                    <input type="checkbox" class="toggle-checkbox" data-tooltip="<%= gettext('Send push notification to mobile devices') %>" checked />
```

Comment:
apologies, the email link brought me right here and I didn't think to look on the Conversation page. and it's super helpful to have!
Please strike the colon at the end of the label. It's not included for other labels in Studio.